### PR TITLE
Added camunda deployment script

### PR DESCRIPTION
**Pull request by Ludo05**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-254

### Change description ###
Added script for automated bpmn and dmn deployment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: scripts/camunda-deployment.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+## Usage: ./camunda-deployment 
+##
+## Options:
+##    - microservice_name: Name of the microservice. Default to `ccd_gw`.
+##
+## deployes bpmn/dmn to camunda.
+
+AUTHORIZATION= sh ./idam-service-token.sh
```
Comment (iguannago) [inline scripts/camunda-deployment.sh]: we have this script already in the /actions folder

File: scripts/idam-service-token.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (iguannago) [inline scripts/idam-service-token.sh]: I think this is a duplicate

**Review by iguannago — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -24,8 +24,13 @@ Since Spring Boot 2.1 bean overriding is disabled. If you want to enable it you
 
 JUnit 5 is now enabled by default in the project. Please refrain from using JUnit4 and use the next generation
 
-## Building and deploying the application
+## Automating BPMN/DMN diagrams
 
+Add the following variables to your bash profile:
+ * `WA_BPMNS_DMNS_PATH` - to the path where you WA BPMNS are stored. [Link to path here](https://github.com/hmcts/wa-standalone-task-bpmn/tree/master/src/main/resources)
```
Comment (BenNzewi) [inline README.md]: small typo; where you WA BPMNS should be where you**r** WA BPMNS

File: README.md [hunk 4]
```diff
@@ -24,8 +24,13 @@ Since Spring Boot 2.1 bean overriding is disabled. If you want to enable it you
...
+ * `IA_TASK_DMNS_BPMNS_PATH` - to the path where DMS in `IA_TASK_CONFIGURATION` are stored. [Link to path here](https://github.com/hmcts/ia-task-configuration/tree/master/src/main/resources)
```
Comment (BenNzewi) [inline README.md]: small typo; to the path where DMS should be to the path where DM**N**S

**Review by alexbaeza — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -24,7 +24,13 @@ Since Spring Boot 2.1 bean overriding is disabled. If you want to enable it you
 
 JUnit 5 is now enabled by default in the project. Please refrain from using JUnit4 and use the next generation
 
-## Building and deploying the application
+## Automating BPMN/DMN diagrams
+
+Add the following variables to your bash profile:
+ * `WA_BPMNS_DMNS_PATH` - to the path where your WA BPMNS are stored. [Link to path here](https://github.com/hmcts/wa-standalone-task-bpmn/tree/master/src/main/resources)
+ * `IA_TASK_DMNS_BPMNS_PATH` - to the path where DMNS in `IA_TASK_CONFIGURATION` are stored. [Link to path here](https://github.com/hmcts/ia-task-configuration/tree/master/src/main/resources)
+
```
Comment (alexbaeza) [inline README.md]: don't need this in this repo

**Review by alexbaeza — CHANGES_REQUESTED** [review]

**Review by alexbaeza — APPROVED** [review]

**Pull request merged**